app/models/certificate.rb

```diff
@@ -0,0 +1,4 @@
+class Certificate < ApplicationRecord
+  validates_presence_of :usage
```

Comment:
we could do `validates_inclusion_of :usage, in: ['signing', 'encryption']` just to start with at least... the actual cert validation is covered by another story

Reply:
validates_inclusion_of: Validates whether the value of the specified attribute is available in a particular enumerable object. 

However I think we want to check whether or not these 2 attributes are null (which is in the scope of this card?).

Reply:
Would it be like this?
validates_inclusion_of :usage, in: ['signing', 'encryption']
validates_presence_of :value, message: 'You must enter certificate value.'

Reply:
the inclusion validates for null (i.e. if it's not one of the options it fails the validation). We should still keep the null validation on `value` on which we'll build with the next story